osdep/macosx_application.m

```diff
@@ -77,7 +82,9 @@
         .macos_title_bar_color = {0, 0, 0, 0},
         .macos_fs_animation_duration = -1,
         .cocoa_cb_sw_renderer = -1,
-        .cocoa_cb_10bit_context = 1
+        .cocoa_cb_10bit_context = 1,
+        // Experimental #HDR on #macOS        
+        .macos_hdr_csp = -1
```

Comment:
there is no -1 in the definition of the option. so you shouldn't se this value here or you need to add it to the option choices. i would add an "auto" option and make this -1.

Reply:
I will add a "none" option, as "auto" should be probably reserved for future functionality where color profile is taken from video header and auto-assigned. How does that sound?

Reply:
none would be wrong though, since it should be the same as the display in that case. also from the doku of the d3d11 omplementation ""auto" is the default, which will select the color space of the desktop on which the swap chain is located.". it should be consistent.

Reply:
OK, got it.